relay-server/src/service.rs

```diff
@@ -142,8 +142,8 @@ impl ServiceState {
         };
 
         let buffer = Arc::new(BufferGuard::new(config.envelope_buffer_size()));
-        let processor = EnvelopeProcessor::start(config.clone(), redis_pool.clone())?;
-        let envelope_manager = EnvelopeManager::create(config.clone(), processor, buffer.clone())?;
+        EnvelopeProcessor::start(config.clone(), redis_pool.clone())?; // TODO(tobias): Registry
```

Comment:
you can do this right now after merging master?  there's no rule saying that you can only have new-style addresses in the new registry...

Reply:
👍 absolutely, makes sense to align that with what's in `master` now.

cc @tobias-wilfert, this would also have to be updated when addressing https://github.com/getsentry/relay/pull/1421#discussion_r949254363

Reply:
Actually, there were unaddressed concerns about the order of initialization with the registry, since it gets added to the `OnceBox` only after the actors already start. If it's alright, I would defer this to @tobias-wilfert for when he addresses the follow-up.